src/mgr/CMakeLists.txt

```diff
@@ -34,7 +34,9 @@ if(WITH_MGR)
   add_executable(ceph-mgr ${mgr_srcs})
   target_link_libraries(ceph-mgr
     osdc client heap_profiler
+    cephsqlite
```

Comment:
might want to conditionalize this on `WITH_LIBCEPHSQLITE` ?

Reply:
ACK, fixed.